cdc/puller/metrics.go

```diff
@@ -53,6 +54,7 @@ var (
 			Help:      "Puller entry buffer size",
 			Buckets:   prometheus.ExponentialBuckets(1, 2, 8),
 		}, []string{"capture", "changefeed"})
+	// todo (Ling Jin): memBufferSize is not used, add it to monitor
```

Comment:
```suggestion
	//TODO(Ling Jin): memBufferSize is not used, add it to monitor
```

Could you file an issue?